src/components/user/SignUpForm/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { View, Text, TouchableHighlight } from 'react-native';
+
+import Input from '../../common/Input';
+import * as constraints from '../../../utils/constraints';
+import styles from './styles';
+import CustomPicker from '../../common/Picker';
+
+const genderOptions = [
+  {
+    label: 'MALE',
+    value: 'male'
+  },
+  {
+    label: 'FEMALE',
+    value: 'female'
+  },
+  {
+    label: 'OTHER',
+    value: 'other'
+  }
+];
+
+const SignUpForm = ({ handleSubmit }) => (
+  <View style={styles.signUp} onSubmit={handleSubmit}>
+    <Field
+      name="username"
+      label="Name"
+      component={Input}
+    />
+    <Field
+      name="email"
+      label="Email"
+      component={Input}
+      keyboardType="email-address"
+    />
+    <Field
+      name="password"
+      label="Password"
+      component={Input}
+      password
+      placeholder="MIN. 8 CHARACTERS LONG"
+    />
+    <Field
+      name="passwordConfirmation"
+      label="Confirm password"
+      component={Input}
+      password
+    />
+    <Field
+      name="gender"
+      label="Gender"
+      component={CustomPicker}
+      options={genderOptions}
+    />
+    <TouchableHighlight onPress={handleSubmit} underlayColor="white">
+      <View style={styles.button}>
```

Comment:
Can't you apply the `button` styles to the `TouchableHighlight` or do you need this View?

Reply:
It's necessary because of the animation when clicking the button. If I do what you are suggesting, while I'm clicking the button it disappears because of the `underlayColor`